common/featured_text.cpp

```diff
@@ -258,6 +258,7 @@ static bool text_tag_init_from_sequence(struct text_tag *ptag,
       ptag->color.background[0] = '\0';
     }
     return true;
+    break;
```

Comment:
break after return? unreachable statement will kick in

Reply:
here is the same and never kicked in:
bool is_player_phase(const struct player *pplayer, int phase)
{
  switch (game.info.phase_mode) {
  case PMT_CONCURRENT:
    return true;
    break;